SetGameTests/ValuesMatchingTest.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CardsMatchingTest.swift
+//  SetGameTests
+//
+//  Created by Artur Shellunts on 12.09.21.
+//
+
+import XCTest
+@testable import SetGame
+
+class ValuesMatchingTest : XCTestCase {
```

Comment:
Consider what Jan said. It's probably an implementation detail of the "CardsMatching" strategy and shouldn't be tested separately.